src/utility/introspectionQueries.js

```diff
@@ -4,14 +4,17 @@
  *  This source code is licensed under the MIT license found in the
  *  LICENSE file in the root directory of this source tree.
  */
+import { getOperationAST, parse, introspectionQuery } from 'graphql';
 
 export { introspectionQuery } from 'graphql';
+export const introspectionQueryName = getOperationAST(parse(introspectionQuery))
+  .name.value;
```

Comment:
Can we guarantee that the `introspectionQuery` from `graphql` will always be named, @IvanGoncharov?

If not, we should add a fallback here:

```suggestion
  .name.value || "IntrospectionQuery";
```

Reply:
> Can we guarantee that the introspectionQuery from graphql will always be named?

Yes, I think we need consistent naming.
I will add `graphql-js` that will check this.

> If not, we should add a fallback here:

You can't do that. If the query doesn't have a name you can't just send `operationName` it will result in an error.

BTW. It's already ~2 years since subscription was added so it would be safe to drop `introspectionQuerySansSubscriptions` in the next major release.

Reply:
> If the query doesn't have a name you can't just send operationName it will result in an error.

Oh yeah, I was thinking of the `introspectionQuerySansSubscriptions` query which we name ourselves, but you're right that we use the same variable with the normal introspection query too.

> safe to drop `introspectionQuerySansSubscriptions` in the next major release.

Noted.

Thanks Ivan!